src/worker/orchestration-execute-result.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
This is not done by Microsoft anymore. We should the Dapr header for this file.

Reply:
license in this repo still points to microsoft, I can regardless update the header in this file

Reply:
Yes, this file, please. We can change the license file to Dapr in the fork as well.